reVX/least_cost_xmission/least_cost_xmission_cli.py

```diff
@@ -148,7 +147,7 @@ def from_config(ctx, config, verbose):
               show_default=True, default=None,
               help=("Number of workers to use for processing, if 1 run in "
                     "serial, if None use all available cores"))
-@click.option('--out_dir', '-o', type=STR, default='./',
+@click.option('--out_dir', '-o', type=STR, default='./out',
```

Comment:
We should probably add a call to `create_dirs` in the function body below to make sure `./out` exists

Reply:
👍